### ISSUE TITLE
Che: Import Fail vertx-paint

### ISSUE THREAD

**Issue by burrsutter** [openshiftio/openshift.io#523]
Import Wizard on 

https://github.com/burrsutter/vertx-paint

```
[STDOUT] Terminal Agent binary is downloaded remotely
[STDOUT] 2017/08/09 14:06:40 Terminal-agent configuration
[STDOUT] 2017/08/09 14:06:40   Server
[STDOUT] 2017/08/09 14:06:40     - Address: :4411
[STDOUT] 2017/08/09 14:06:40     - Base path: ''
[STDOUT] 2017/08/09 14:06:40   Terminal
[STDOUT] 2017/08/09 14:06:40     - Slave command: ''
[STDOUT] 2017/08/09 14:06:40     - Activity tracking enabled: false
[STDOUT] 2017/08/09 14:06:40 
[STDOUT] 2017/08/09 14:06:40 ⇩ Registered HTTPRoutes:
[STDOUT] 
[STDOUT] 2017/08/09 14:06:40 Terminal routes:
[STDOUT] 2017/08/09 14:06:40 ✓ Connect to pty(webscoket) ............... GET    /pty
[STDOUT] 2017/08/09 14:06:40 
[STDOUT] Exec Agent binary is downloaded remotely
[STDOUT] 2017/08/09 14:06:48 Exec-agent configuration
[STDOUT] 2017/08/09 14:06:48   Server
[STDOUT] 2017/08/09 14:06:48     - Address: :4412
[STDOUT] 2017/08/09 14:06:48     - Base path: ''
[STDOUT] 2017/08/09 14:06:48   Process executor
[STDOUT] 2017/08/09 14:06:48     - Logs dir: /workspace-logs/dev-machine-exec-agent
[STDOUT] 2017/08/09 14:06:48 
[STDOUT] 2017/08/09 14:06:48 ⇩ Registered HTTPRoutes:
[STDOUT] 
[STDOUT] 2017/08/09 14:06:48 Process Routes:
[STDOUT] 2017/08/09 14:06:48 ✓ Start Process ........................... POST   /process
[STDOUT] 2017/08/09 14:06:48 ✓ Get Process ............................. GET    /process/:pid
[STDOUT] 2017/08/09 14:06:48 ✓ Kill Process ............................ DELETE /process/:pid
[STDOUT] 2017/08/09 14:06:48 ✓ Get Process Logs ........................ GET    /process/:pid/logs
[STDOUT] 2017/08/09 14:06:48 ✓ Get Processes ........................... GET    /process
[STDOUT] 2017/08/09 14:06:48 
[STDOUT] 2017/08/09 14:06:48 Exec-Agent WebSocket routes:
[STDOUT] 2017/08/09 14:06:48 ✓ Connect to Exec-Agent(websocket) ........ GET    /connect
[STDOUT] 2017/08/09 14:06:48 
[STDERR] chown: changing ownership of ‘/projects’: Operation not permitted
[STDOUT] 2017/08/09 14:06:48 ⇩ Registered RPCRoutes:
[STDOUT] 
[STDOUT] 2017/08/09 14:06:48 Process Routes:
[STDOUT] 2017/08/09 14:06:48 ✓ process.start
[STDOUT] 2017/08/09 14:06:48 ✓ process.kill
[STDOUT] 2017/08/09 14:06:48 ✓ process.subscribe
[STDOUT] 2017/08/09 14:06:48 ✓ process.unsubscribe
[STDOUT] 2017/08/09 14:06:48 ✓ process.updateSubscriber
[STDOUT] 2017/08/09 14:06:48 ✓ process.getLogs
[STDOUT] 2017/08/09 14:06:48 ✓ process.getProcess
[STDOUT] 2017/08/09 14:06:48 ✓ process.getProcesses
[STDERR] chown: changing ownership of ‘/projects’: Operation not permitted
[STDERR] chmod: changing permissions of ‘/projects’: Operation not permitted
[STDOUT] Workspace Agent will be downloaded from Workspace Master
[STDOUT] Listening for transport dt_socket at address: 4403
[STDOUT] 2017-08-09 14:06:57,831[main]             [INFO ] [o.a.c.s.VersionLoggerListener 89]    - Server version:        Apache Tomcat/8.5.11
[STDOUT] 2017-08-09 14:06:57,835[main]             [INFO ] [o.a.c.s.VersionLoggerListener 91]    - Server built:          Jan 10 2017 21:02:52 UTC
[STDOUT] 2017-08-09 14:06:57,835[main]             [INFO ] [o.a.c.s.VersionLoggerListener 93]    - Server number:         8.5.11.0
[STDOUT] 2017-08-09 14:06:57,836[main]             [INFO ] [o.a.c.s.VersionLoggerListener 95]    - OS Name:               Linux
[STDOUT] 2017-08-09 14:06:57,836[main]             [INFO ] [o.a.c.s.VersionLoggerListener 97]    - OS Version:            3.10.0-514.25.2.el7.x86_64
[STDOUT] 2017-08-09 14:06:57,836[main]             [INFO ] [o.a.c.s.VersionLoggerListener 99]    - Architecture:          amd64
[STDOUT] 2017-08-09 14:06:57,837[main]             [INFO ] [o.a.c.s.VersionLoggerListener 101]   - Java Home:             /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.141-1.b16.el7_3.x86_64/jre
[STDOUT] 2017-08-09 14:06:57,837[main]             [INFO ] [o.a.c.s.VersionLoggerListener 103]   - JVM Version:           1.8.0_141-b16
[STDOUT] 2017-08-09 14:06:57,837[main]             [INFO ] [o.a.c.s.VersionLoggerListener 105]   - JVM Vendor:            Oracle Corporation
[STDOUT] 2017-08-09 14:06:57,838[main]             [INFO ] [o.a.c.s.VersionLoggerListener 107]   - CATALINA_BASE:         /home/user/che/ws-agent
[STDOUT] 2017-08-09 14:06:57,838[main]             [INFO ] [o.a.c.s.VersionLoggerListener 109]   - CATALINA_HOME:         /home/user/che/ws-agent
[STDOUT] 2017-08-09 14:06:57,839[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Dnop
[STDOUT] 2017-08-09 14:06:57,840[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
[STDOUT] 2017-08-09 14:06:57,840[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -XX:+UseG1GC
[STDOUT] 2017-08-09 14:06:57,840[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -XX:+UseStringDeduplication
[STDOUT] 2017-08-09 14:06:57,840[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -XX:MinHeapFreeRatio=20
[STDOUT] 2017-08-09 14:06:57,841[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -XX:MaxHeapFreeRatio=40
[STDOUT] 2017-08-09 14:06:57,841[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -XX:MaxRAM=1300m
[STDOUT] 2017-08-09 14:06:57,845[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Xms256m
[STDOUT] 2017-08-09 14:06:57,845[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Dche.logs.dir=/workspace-logs/dev-machine-ws-agent
[STDOUT] 2017-08-09 14:06:57,846[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Dche.logs.level=INFO
[STDOUT] 2017-08-09 14:06:57,846[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Djuli-logback.configurationFile=file:/home/user/che/ws-agent/conf/tomcat-logger.xml
[STDOUT] 2017-08-09 14:06:57,846[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Djdk.tls.ephemeralDHKeySize=2048
[STDOUT] 2017-08-09 14:06:57,847[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Djava.protocol.handler.pkgs=org.apache.catalina.webresources
[STDOUT] 2017-08-09 14:06:57,847[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -agentlib:jdwp=transport=dt_socket,address=4403,server=y,suspend=n
[STDOUT] 2017-08-09 14:06:57,848[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Dcom.sun.management.jmxremote
[STDOUT] 2017-08-09 14:06:57,848[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Dcom.sun.management.jmxremote.ssl=false
[STDOUT] 2017-08-09 14:06:57,848[main]             [INFO ] [o.a.c.s.VersionLoggerListener 115]   - Command line argument: -Dcom.sun.management.jmxremote.authenticate=false
[STDOUT] Terminal Agent binary is downloaded remotely
[STDOUT] 2017/08/09 14:13:49 Terminal-agent configuration
[STDOUT] 2017/08/09 14:13:49   Server
[STDOUT] 2017/08/09 14:13:49     - Address: :4411
[STDOUT] 2017/08/09 14:13:49     - Base path: ''
[STDOUT] 2017/08/09 14:13:49   Terminal
[STDOUT] 2017/08/09 14:13:49     - Slave command: ''
[STDOUT] 2017/08/09 14:13:49     - Activity tracking enabled: false
[STDOUT] 2017/08/09 14:13:49 
[STDOUT] 2017/08/09 14:13:49 ⇩ Registered HTTPRoutes:
[STDOUT] 
[STDOUT] 2017/08/09 14:13:49 Terminal routes:
[STDOUT] 2017/08/09 14:13:49 ✓ Connect to pty(webscoket) ............... GET    /pty
[STDOUT] 2017/08/09 14:13:49 
[STDOUT] Exec Agent binary is downloaded remotely
[STDOUT] 2017/08/09 14:13:57 Exec-agent configuration
[STDOUT] 2017/08/09 14:13:57   Server
[STDOUT] 2017/08/09 14:13:57     - Address: :4412
[STDOUT] 2017/08/09 14:13:57     - Base path: ''
[STDOUT] 2017/08/09 14:13:57   Process executor
[STDOUT] 2017/08/09 14:13:57     - Logs dir: /workspace-logs/dev-machine-exec-agent
[STDOUT] 2017/08/09 14:13:57 
[STDOUT] 2017/08/09 14:13:57 ⇩ Registered HTTPRoutes:
[STDOUT] 
[STDOUT] 2017/08/09 14:13:57 Process Routes:
[STDOUT] 2017/08/09 14:13:57 ✓ Start Process ........................... POST   /process
[STDOUT] 2017/08/09 14:13:57 ✓ Get Process ............................. GET    /process/:pid
[STDOUT] 2017/08/09 14:13:57 ✓ Kill Process ............................ DELETE /process/:pid
[STDOUT] 2017/08/09 14:13:57 ✓ Get Process Logs ........................ GET    /process/:pid/logs
[STDOUT] 2017/08/09 14:13:57 ✓ Get Processes ........................... GET    /process
[STDOUT] 2017/08/09 14:13:57 
[STDOUT] 2017/08/09 14:13:57 Exec-Agent WebSocket routes:
[STDOUT] 2017/08/09 14:13:57 ✓ Connect to Exec-Agent(websocket) ........ GET    /connect
[STDOUT] 2017/08/09 14:13:57 
[STDERR] chown: changing ownership of ‘/projects’: Operation not permitted
[STDOUT] 2017/08/09 14:13:57 ⇩ Registered RPCRoutes:
[STDOUT] 
[STDOUT] 2017/08/09 14:13:57 Process Routes:
[STDOUT] 2017/08/09 14:13:57 ✓ process.start
[STDOUT] 2017/08/09 14:13:57 ✓ process.kill
[STDOUT] 2017/08/09 14:13:57 ✓ process.subscribe
[STDOUT] 2017/08/09 14:13:57 ✓ process.unsubscribe
[STDOUT] 2017/08/09 14:13:57 ✓ process.updateSubscriber
[STDOUT] 2017/08/09 14:13:57 ✓ process.getLogs
[STDOUT] 2017/08/09 14:13:57 ✓ process.getProcess
[STDOUT] 2017/08/09 14:13:57 ✓ process.getProcesses
[STDERR] chown: changing ownership of ‘/projects’: Operation not permitted
[STDERR] chmod: changing permissions of ‘/projects’: Operation not permitted
[STDOUT] Workspace Agent will be downloaded from Workspace Master
```

**Comment by joshuawilson** [comment]
@burrsutter where did you run the Import? In Che or in the app wizard?

**Comment by ibuziuk** [comment]
@burrsutter is it still reproducible?

**Comment by l0rd** [comment]
I have verified that this is fixed now (che v2.0.31). @burrsutter feel free to reopen if you can reproduce it.
